client/client.go

```diff
@@ -165,15 +167,12 @@ func (c *Client) SearchByLabels(ctx context.Context, opts ...GetOption) ([]*mode
 	if options.Project == "" {
 		options.Project = defaultProject
 	}
-	lableReq := "q="
-	i := 1
+	lableReq := ByLabelsPrefix
```

Comment:
ByLabelsPrefix -> QueryParamQ

the convention is use http param "query param","header" as const prefix